mapper/mapper.go

```diff
@@ -435,13 +443,36 @@ func (m *Mapper) graphqlSpecialTypes(field *graphql.Field, protoTypeName string)
 	return field
 }
 
-func (m *Mapper) buildOneofMapper(oneof *descriptor.Oneof, input bool) *OneofMapper {
+func (m *Mapper) buildOneofMapper(oneof *descriptor.Oneof, input bool, unionTypeName string, existedMapper *OneofMapper) *OneofMapper {
```

Comment:
🙏 please add a comment on what we changed and why. It could help to merge any conflicts with upstream in the future for other persons.